modules/jenkins-plugin/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>2.2</version>
+    </parent>
+    <properties>
+        <!-- Users must have at least this Jenkins version to use this plugin. -->
+        <jenkins.version>1.609.1</jenkins.version>
+        <java.level>8</java.level>
+    </properties>
+
+    <groupId>io.swagger.assert</groupId>
+    <artifactId>assert4j-jenkins-plugin</artifactId>
+    <name>Swagger Assert4J Jenkins Plugin</name>
```

Comment:
@spindelmanne how hard would it be to move the jenkins plugin to a separate repo? are there any internal dependencies we need to consider? perhaps the maven-plugin should be part of that repo also since they are both more related to TestServer than to local execution